extensions/guidepost/README.md

```diff
@@ -0,0 +1,40 @@
+# Extension - Guidepost
+## Description
+Used to set guidepost objects onto the canvas.
+The config file can be found at [/run/map/watermark/guideposts.json](../../run/map/watermark/guideposts.json)
+
+## Parameters
+* assetName: the guidepost asset name being called
+* src: path to the asset
+* imageRef: image reference
+* The following parameter can refer to [CanvasRenderingContext2D.drawImage](https://developer.mozilla.org/zh-CN/docs/Web/API/CanvasRenderingContext2D/drawImagehttps://developer.mozilla.org/zh-CN/docs/Web/API/CanvasRenderingContext2D/drawImage) function
+    * srcX: sx. the coordinate X in the image to start from
+    * srcY: sy. the coordinate Y in the image to start from
+    * sWidth: sWidth. the width of image to be used
+    * sHeight: sHeight. the height of image to be used
+    * dWidth: dWidth. the width of image to be zoomed
+    * dHeight: dHeight. the height of image to be zoomed
+* MapCoord: Refer to [common/maplib/map.mjs](../../common/maplib/map.mjs)
+* position: 1~9. the position that the image should be aligned
```

Comment:
I think that using string literals is more readable than using integer constants.
```
x,y
top, left, bottom, right
topleft, bottomleft, topright, bottomright
midtop, midleft, midbottom, midright
center, centerx, centery
size, width, height
w,h
```
These are copied from the [documentation of Pygame](https://www.pygame.org/docs/ref/rect.html) for your information.

Reply:
I changed directions into the literal ones, but I think x, y, w, h are not necessary to change, because I think srcX, srcY, sWidth, ...... can be more understandable.